cmd/http/config.json

```diff
@@ -1,26 +1,35 @@
 {
     "ConsistentHash": "rendezvous",
     "BucketSize": 10,
-    "ReplicaNum": 3,
+    "ReplicaNum": {
+        "//" : "The number of local replica from the ones whose latency is fewer than RemoteStoreLatencyThresholdInMilliSec", 
+        "Local": 2,
```

Comment:
can these numbers (local being 2 and remote being 1) be calculated in the code based on the store configuration and the threshold?

Reply:
No. We discussed before. For low latencies, we try to write fewer local stores. However, it might cause uneven distribution. For even distribution, we might need to write many stores, which might cause high latency.  Something like CAP, we can't get both low latency and even distribution. The thing here is to find a balance point.  The first step is to manually configured. Next we can add policy or even learning model to get the best result.